config/application.rb

```diff
@@ -41,6 +41,11 @@ class Application < Rails::Application
     # the I18n.default_locale when a translation can not be found)
     config.i18n.fallbacks = true
 
+    # For deployment through an iframe as part of the europeana portal, the application needs to load a JS file
+    # The value provided here is not a stable version, you should specify a fixed version in:
+    # config/initializers/e7a_1418_iframe.rb
+    config.e7a_1418_iframe_source = 'http://styleguide.europeana.eu/js/modules/eu/e7a_1418_iframe.js'
```

Comment:
I considered this, but I think this way we have more control should we ever need to refactor where the .js file lives within the styleguide, or rename it. By supplying the full path we don't need to change any code in the application to update it.